source/more-info/unsupported/systemd_journal.markdown

```diff
@@ -32,5 +32,20 @@ sudo apt-get install --reinstall systemd-journal-remote
 
 and reboot your system
 
+## Alternative solution
```

Comment:
That is not an alternative solution, but should maybe be an integral part of the text above; ensuring it is running (and also enabled, which IMHO is missed in the added content).